src/app/pages/blue-sky/style.scss

```diff
@@ -340,6 +345,15 @@ $mobile-padding: 30px;
         margin-bottom: 2rem;
         font-weight: 300;
         text-align: center;
+
+        a {
+          text-decoration: none;
+          color: inherit;
+
+          &:hover {
+            text-decoration: underline;
```

Comment:
Link buttons don't receive underlines [per Bluebird UI](https://bluebirdui.com/?path=/docs/components-buttons--link-button)

Reply:
That class only applied to the footer links (not buttons) in this instance.